sdk/automation/Azure.ResourceManager.Automation/src/Generated/Models/AutomationErrorResponse.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.Automation.Models
+{
+    /// <summary>
+    /// Error response of an operation failure
+    /// Serialized Name: ErrorResponse
+    /// </summary>
+    public partial class AutomationErrorResponse
```

Comment:
In Azure.Core we defined the common type `ResponseError`, so here I think we should rename this one to `AutomationResponseError` to keep consistent.

Reply:
fixed